src/Files.App/Constants.cs

```diff
@@ -124,16 +124,13 @@ public static class GridViewBrowser
 			{
 				public const int GridViewIncrement = 20;
 
-				// Max achievable ctrl + scroll, not a default layout size
-				public const int GridViewSizeMax = 300;
```

Comment:
I removed `GridViewSizeMax` because `220` is the largest size that we know will always have a thumbnail.